sentences/fr/homeassistant_HassGetCurrentDate.yaml

```diff
@@ -12,5 +12,5 @@ intents:
           - "Nous sommes le combien[ aujourd'hui]"
           - "C'est quoi la date[ aujourd'hui]"
           - "C'est quoi la date du jour"
-          - "Quel jour est-il"
-          - "Quel jour est il"
+          - "C'est quel jour[ aujourd'hui]"
+          - "Quel jour est(-| )il"
```

Comment:
Please keep it a 2 separate sentences as it can be hard for beginner to understand this syntax.